### PR TITLE
Deploy to production

### PR DESCRIPTION
**Pull request by RafaelEchart**
For this Capstone project @Amrhub and @RafaelEchart did the following:

- Get a list of meals with a unique ID for each meal according to the selected category [From the Meals API](https://www.themealdb.com/api.php)

- For a given meal you can get detailed information using the meal ID.

- We used the [Involvement API](https://www.notion.so/microverse/Involvement-API-869e60b5ad104603aa6db59e08150270) to record the different user interactions (likes, comments).

- We built these interfaces:
     - The home page.
     - The comments popup.
 
- We followed the layout of the wireframes provided. We personalized the rest of the design including colors, typographies, spacings, etc

###  Home page
- When the page loads, the web app retrieves data from:
   - The selected API and shows the list of items on the screen.
   - The Involvement API to show the item likes.
   - When the user clicks on the Like button of an item, the interaction is recorded in the Involvement API and the screen is updated.
   - When the user clicks on the "Comments" button, the Comments popup appears.
   - Home page header and navigation are similar to the given mockup.-
   - Home page footer is similar to the given mockup.

### Comments popup
- When the popup loads, the web app retrieves data from:
    - The selected API and shows details about the selected item.
    - The Involvement API to show the item comments.
- When the user clicks on the "Comment" button, the data is recorded in the Involvement API and the screen is updated.

### Counters:

- The number of items (home).
- The number of comments (comments popup).

### Tests

- We used Jest to test:
    - Function that returns the number of items on the Home page
    - Function that returns the number of comments on the Comment Popup

### Project documentation

[Capstone Overview](https://drive.google.com/file/d/1tcHf2zilMFW-HCkPU4PMbI2r1AYlN-O1/view)

### Workload distribution

[Kanban Board](https://github.com/RafaelEchart/RestaurantMenu_Capstone/projects/1)

### Additional features

- The user can only like once for each meal.
- Responsive design
- Display loader for each request (User feedback)
- Transition for like button and comment popup

### [Project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md#project-requirements)

### CODE REVIEW

File: src/styles/popUpWindow.css [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+  border-radius: 20px;
+  cursor: pointer;
+  background-color: var(--primary-color);
+}
+
+.popup-window .comments-form {
+  gap: 0.5rem;
+  width: 100%;
+  max-width: 500px;
+}
```
Comment (henatan99) [inline src/styles/popUpWindow.css:67]: - [ ] Everything looks great. You did an amazing job in this project fulfilling almost all the project requirements impressively. Just for the sake of strictly complying with the design requirements, I request you to make a very slight change. The alignment of this component is not as in the original design. Just take this as a business requirement given be a client and make changes accordingly. :pray: 

image

image

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by RafaelEchart** [comment]
As the previous CR suggested the layout should be all aligned to the left, this includes: 

- Description box
- comment section
- new comment inputs

**Review by rukundoeric — APPROVED** [review]
Hi @RafaelEchart,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**